packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -138,12 +142,42 @@ function updateLi(node: MbNode, iconAttributes: IconAttributes) {
 // See https://github.com/MarkBind/markbind/pull/2316#discussion_r1255364486 for more details.
 function handleLiNode(node: MbNode, iconAttrValue: IconAttributeDetail) {
   if (iconAttrValue.isFirst) {
-    iconAttrValue.iconAttrs = getIconAttributes(node);
+    const nodeIconAttrs = getIconAttributes(node);
+    // Check if first item is customized
+    if (nodeIconAttrs?.icon !== undefined) {
+      iconAttrValue.addIcons = true;
+    }
+    // Save if the icon is not one-off
+    if (!nodeIconAttrs?.oneOff) {
+      iconAttrValue.iconAttrs = nodeIconAttrs;
+    }
     iconAttrValue.isFirst = false;
   } else if (iconAttrValue.iconAttrs) {
-    iconAttrValue.iconAttrs = getIconAttributes(node, iconAttrValue.iconAttrs);
+    const nodeIconAttrs = getIconAttributes(node, iconAttrValue.iconAttrs);
+    // Save if there is icon and not one-off
+    if (nodeIconAttrs?.icon !== undefined && !nodeIconAttrs?.oneOff) {
```

Comment:
What about the other attributes? like if I want to define `i-size` as oneOff will this still work?

Reply:
My impression was that the one-off will apply to the entire customization.

Reply:
I have implemented such that it applies to the whole icon, starting from the presence of an icon, since there would be no other attributes if there is no icon.  It would be useful for it to be targetted if you can overwrite only specific attributes rather than the whole icon.

Reply:
It is currently implemented to apply to entire customization.